### DISCUSSION TITLE
How to build for macOS Monterey

### DISCUSSION THREAD

**Discussion by GitPuffy** [hmatuschek/qdmr#198] [Q&A]
I see that the last macOS DMG build was for 0.8.x.  How do I build for macOS one of the more recent releases?

Thank you.

**Comment by hmatuschek** [comment]
If you actually want to build it from sources, you can follow the build instructions in https://dm3mat.darc.de/qdmr/install.html. The dependencies can be installed using MacPorts (https://www.macports.org/). However, the recommended approach is to install qdmr using MacPorts directly (https://ports.macports.org/port/qdmr/details/).

**Reply by GitPuffy** [reply]
I did just that, installed MacPorts and have a successful build on Monterey.

**Reply by GitPuffy** [reply]
Is there a way to run the build process but pulling down the current commits instead of a release?  I'd like to try out the changes made for dark mode, for example.

**Reply by hmatuschek** [reply]
Sure, either you clone the master branch using git i.e., 
```
git clone https://github.com/hmatuschek/qdmr.git
```
or you download the [current master as a zip file](https://github.com/hmatuschek/qdmr/archive/refs/heads/master.zip).

**Comment by bderry71** [comment]
Is there a workflow process on Github that can compile and build the App bundle for macOS as you tag releases? That way us macOS folks can just get the app bundle when there is a new one and not have to load MacPorts on our machines.
